File: Python/Manual 02/Question02.py
#!/usr/bin/env python3


def case_Add(num1, num2):
    return num1 + num2


def case_Subt(num1, num2):
    return num1 - num2


def case_Mult(num1, num2):
    return num1 * num2


def case_Div(num1, num2):
    return num1 / num2


def case_Default():
    print("Invalid Operation")



symbol = str(input("Enter an Arithmetic Symbol: "))
num1 = float(input("Enter First Number: "))
num2 = float(input("Enter Second Number: "))

case = {"+": case_Add(num1, num2), "-": case_Subt(num1, num2), "*": case_Mult(num1, num2), "/": case_Div(num1, num2)}

if len(symbol) > 1 or not (
    symbol == "+" or symbol == "-" or symbol == "*" or symbol == "/"
):
    print("Enter a Valid Operation")
else:
    print(case.get(symbol, case_Default))

File: JavaScript/Manual 02/Question02.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter an Arithmetic Operator: ", function (operator) {
    rl.question("Enter First Number: ", function (firstNumber) {
        rl.question("Enter Second Number: ", function (secondNumber) {
            switch (operator) {
                case '+':
                    console.log("The Sum of " + firstNumber + " and " + secondNumber + " is: " + (parseFloat(firstNumber) + parseFloat(secondNumber)));
                    break;
                case '-':
                    console.log("The Difference of " + firstNumber + " and " + secondNumber + " is: " + (parseFloat(firstNumber) - parseFloat(secondNumber)));
                    break;
                case '*':
                    console.log("The Product of " + firstNumber + " and " + secondNumber + " is: " + (parseFloat(firstNumber) * parseFloat(secondNumber)));
                    break;
                case '/':
                    console.log("The Division of " + firstNumber + " and " + secondNumber + " is: " + (parseFloat(firstNumber) / parseFloat(secondNumber)));
                    break;
                case '%':
                    if (secondNumber == 0) {
                        console.log("Cannot Divide by Zero");
                        break;
                    }
                    console.log("The Modulus of " + firstNumber + " and " + secondNumber + " is: " + (parseFloat(firstNumber) % parseFloat(secondNumber)));
                    break;
                default:
                    console.log("Invalid Operator");
            }
            rl.close();
        })
    })
})